CMakeLists.txt

```diff
@@ -1464,6 +1489,7 @@ _add_if("HTTP3"         USE_NGTCP2 OR USE_QUICHE)
 _add_if("MultiSSL"      CURL_WITH_MULTI_SSL)
 _add_if("HTTPS-proxy"   SSL_ENABLED AND (USE_OPENSSL OR USE_GNUTLS OR USE_NSS))
 _add_if("unicode"       ENABLE_UNICODE)
+_add_if("threadsafe-init" (NOT WIN32 AND HAVE_ATOMIC) OR (HAVE_SRW_LOCK))
```

Comment:
This feature has been renamed to `threadsafe` while this PR has been in the works!